python/python/glide/async_commands/transaction.py

```diff
@@ -2218,6 +2218,51 @@ def zdiffstore(
             RequestType.ZDiffStore, [destination, str(len(keys))] + keys
         )
 
+    def zinter(
+        self: TTransaction,
+        keys: List[str],
+    ) -> TTransaction:
+        """
+        Computes the intersection of sorted sets given by the specified `keys` and returns a list of intersecting elements.
+
+        When in cluster mode, all keys in `keys` must map to the same hash slot.
```

Comment:
We don't add this notice to transaction
https://github.com/aws/glide-for-redis/pull/1476#discussion_r1617656567
```suggestion
```